src/main/java/com/kodilla/checkers/gamelogic/BoardPosition.java

```diff
@@ -0,0 +1,91 @@
+package com.kodilla.checkers.gamelogic;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.Math.abs;
+
+public class BoardPosition {
+
+    private final int x;
+    private final int y;
+    public List<BoardPosition> route;
```

Comment:
Czy tutaj chodzi o ruch wielokrotny? Jeśli tak, to to jest trudna abstrakcja - nie będzie łatwiej (i prościej dla gracza) zaimplementować to tak, że po wykonaniu bicia po prostu jeśli są możliwości, to możesz wykonać tak jakby kolejny ruch.